src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -648,26 +651,27 @@ public static boolean verifyBag(final File bagDir) {
     /**
      * Method to find and set the repository root from the resource uri.
      * @param uri the URI for the resource
-     * @throws IOException
-     * @throws FcrepoOperationFailedException
      */
-    private void findRepositoryRoot(final URI uri) {
-        repositoryRoot = uri;
+    @VisibleForTesting
+    public URI findRepositoryRoot(final URI uri) {
```

Comment:
You can likely make the `protected` if the unit test is moved to `ImporterTest.java`.